main/ertp/api/issuer.md

```diff
@@ -1,214 +1,252 @@
+useful for debugging and double-checking assumptions, but should not be trusted.
 
-The `mathHelpersName` will be used to import a specific mathHelpers from the mathHelpers library. For example, `natMathHelpers`, the default, supports
-natural numbers and is used for basic fungible tokens.
+The optional `amountMathKind` specifies which of three kinds the created `amountMath` is.
+Each implements all of the same set of API methods (i.e. `amountMath` methods are 
+polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` 
+instead of making the strings yourself.
+- `MathKind.NAT` (`nat`): Used with fungible assets. amount values are natural numbers (non-negative integers). Default value.
+- `MathKind.STRING_SET` (`strSet`): Used with non-fungible assets. amount values are strings.
+- `MathKind.SET` (`set`): Used with non-fungible assets. amount values are objects or records with multiple properties.
 
 ```js
-const { issuer, mint, amountMath } = makeIssuerKit('bucks');
+import { MathKind, makeIssuerKit } from '@agoric/ertp';
+makeIssuerKit('quatloos`); // Defaults to 'MathKind.NAT'
+makeIssuerKit('foobars', 'MathKind.STRSET');
+makeIssuerKit('kitties', MathKind.SET');
+```
+
+```js
+const { issuer: quatloosIssuer, mint: quatloosMint, amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
 // This is merely an amount, describing assets.
-const bucks2 = amountMath.make(2);
+const quatloos2 = quatloosAmountMath.make(2);
 
-const { mint, issuer, amountMath } = makeIssuerKit('alamedaCountyPropertyTitle', 'strSet');
+const { mint: titleMint, issuer: titleIssuer, amountMath: titleAmountMath } = makeIssuerKit('alamedaCountyPropertyTitle', 'strSet');
 
 // These are merely amounts describing digital assets, not minting assets.
-const cornerProperty = amountMath.make(harden['1292826']);
-const adjacentProperty = amountMath.make(harden['1028393']);
-const combinedProperty = amountMath.make(harden['1292826', '1028393']);
+const cornerProperty = titleAmountMath.make(harden['1292826']);
+const adjacentProperty = titleAmountMath.make(harden['1028393']);
+const combinedProperty = titleAmountMath.make(harden['1292826', '1028393']);
 ```
 
 ## issuer.getBrand()
-- Returns: `{Brand}` - The brand for the issuer.
+- Returns: `{Brand}` 
 
-Get the Brand for this Issuer. The Brand indicates the kind of digital asset and is shared by the mint, the issuer, and any purses and payments of this particular kind. The brand is not closely held, so this function should not be trusted to identify an issuer alone. Fake digital assets and amounts can use another issuer's brand.
+Returns the `brand` for this `issuer`. The `brand` indicates the kind of digital asset
+and is the same for the `issuer`'s associated `mint`, and any `purses` and `payments` of this particular
+kind. The `brand` is not closely held, so this function should not be trusted to identify
+an `issuer` alone. Fake digital assets and amounts can use another `issuer's `brand`.
```

Comment:
Honestly, I think it might just make it more confusing. The API docs don't really mention or explain vats or how we're using them. I think this also might fit best in the JavaScript Distributed Programming doc, aka tips on how to program with some behind the scenes implementation details like this to explain.